pkg/shares/non_interactive_defaults.go

```diff
@@ -80,3 +82,16 @@ func roundUpBy(cursor, v int) int {
 		return ((cursor / v) + 1) * v
 	}
 }
+
+// MinSquareSize returns the minimum square size that can contain shareCount
+// number of shares.
+//
+// NOTE: this function was duplicated to avoid import cycles.
```

Comment:
[non-blocking question]
where's the other one?

Reply:
https://github.com/celestiaorg/celestia-app/blob/b65353f456335ed195c67522d4ec546dc3a583fb/x/blob/types/payforblob.go#L178-L182